src/main/webapp/js/feedbackResponseComments.js

```diff
@@ -502,6 +506,14 @@ function loadFeedbackResponseComments(user, courseId, fsName, fsIndx, clickedEle
     
 }
 
+function deleteResponseComment(callback) {
+    BootboxWrapper.showModalConfirmation('Warning', 'Are you sure you want to remove this comment ?', function(result) {
+        if (result) {
+            callback();
+        }
```

Comment:
I think there is no need to check for the `result`, you can pass in the `callback` as the third argument directly (the `okCallback`).

Also, since it's a warning, you should pass in `StatusType.WARNING` as the last argument (which unfortunately means you'll have to pass in the default values for the arguments in between)

Something like:
~~~
.showModalConfirmation('Warning', 'You sure?', callback, null,
        BootboxWrapper.DEFAULT_OK_TEXT, BootboxWrapper.DEFAULT_CANCEL_TEXT,
        StatusType.WARNING);